### ISSUE TITLE
input.range: orientation attribuite doesprograms properly

### ISSUE THREAD

**Issue by consultit** [mikke89/RmlUi#61]
Hi, I'm just experimenting with Samples programs, and it seems that range's orientation attribuite, if specified as both horizontal and vertical, displays  the trackbar collapsed with only + and - images visibile.

**Comment by mikke89** [comment]
You're right, thanks! The orientation attribute wasn't detected properly and it always set it to vertical. It looked collapsed because it isn't styled for vertical. 

Hopefully it works for you now, try adding a larger height when using vertical mode (although it might look a bit off).

**Comment by mikke89** [comment]
Great!
